src/vstest.console/CommandLine/Executor.cs

```diff
@@ -222,7 +222,14 @@ private int GetArgumentProcessors(string[] args, out List<IArgumentProcessor> pr
         // Examples: processors to enable loggers that are statically configured, and to start logging,
         // should always be executed.
         var processorsToAlwaysExecute = processorFactory.GetArgumentProcessorsToAlwaysExecute();
-        processors.AddRange(processorsToAlwaysExecute);
+        foreach (var processor in processorsToAlwaysExecute)
+        {
+            if (processors.Any(i => i.Metadata.Value.CommandName == processor.Metadata.Value.CommandName)) {
+                continue;
+            }
+
+            processors.Add(ArgumentProcessorFactory.WrapLazyProcessorToInitializeOnInstantiation(processor));
```

Comment:
```suggestion
            // We need to initialize the argument executor if it's set to always execute. This ensures it will be initialized with other executors.
            processors.Add(ArgumentProcessorFactory.WrapLazyProcessorToInitializeOnInstantiation(processor));
```